collectors/proc.plugin/README.md

```diff
@@ -462,6 +463,48 @@ and metrics:
     the corresponding `min` or `empty`, which will then always read as zero.
     This way, alerts which match on these will still work.
 
+## Infiniband interconnect
+
+This module monitors every active Infiniband port. It provides generic counters statistics, and per-vendor hw-counters (if vendor is supported).
+
+### Monitored interface metrics
+
+Each port will have its counters metrics monitored, grouped in the following charts:
+
+-   **Bandwidth usage**
+    Sent/Received data, in KB/s
+
+-   **Packets Statistics**
+    Sent/Received packets, in 3 categories: total, unicast and multicast.
+
+-  **Errors Statistics**
+    Many errors counters are provided, presenting statistics for:
+    - Packets: malformated, sent/received discarded by card/switch, missing ressource
+    - Link: downed, recovered, integrity error, minor error
+    - Other events: Tick Wait to send, buffer overrun
+
+If your vendor is supported, you'll also get HW-Counters statistics. These being vendor specific, please report to their documentation.
```

Comment:
Did you mean `refer` instead of `report`?

Reply:
Indeed, it was a bad translation from french, where we say "reportez vous à"
thanks